amplify/backend/function/dogfunction/src/event.json

```diff
@@ -0,0 +1,5 @@
+{
+    "key1": "value1",
```

Comment:
can you explain this to me @ZhuochaoZhong

Reply:
This is the test case generated by Lambda, you change that for testing. Since I did most of the test through the aws console, we could ignore this file.